ci-operator/config/openshift/elasticsearch-operator/openshift-elasticsearch-operator-master.yaml

```diff
@@ -63,7 +63,7 @@ build_root:
 images:
 - from: base
   inputs:
-    ocp_builder_rhel-8-golang-1.19-openshift-4.12:
+    ocp_builder_rhel-9-golang-1.20-openshift-4.14:
```

Comment:
This won't work if you do not introduce it in the images section first:
```yaml
  ocp_builder_rhel-9-golang-1.20-openshift-4.14:
    name: builder
    namespace: ocp
tag: rhel-9-golang-1.20-openshift-4.14
```

Reply:
Yeah I messed up, I reverted everything when I realised I was also updating the ES image and not only the operator image and forgot this

Reply:
Do we need this one at all then:
```yaml
  ocp_builder_rhel-8-golang-1.19-openshift-4.12:
    name: builder
    namespace: ocp
    tag: rhel-8-golang-1.19-openshift-4.12
```